pymeasure/display/inputs.py

```diff
@@ -41,10 +41,20 @@ def __init__(self, parameter):
         super().__init__()
         self._parameter = None
         self.set_parameter(parameter)
+        self.set_value(parameter)
 
     def set_parameter(self, parameter):
         self._parameter = parameter
 
+    def set_value(self, parameter):
+        if hasattr(self, 'setValue'):
+            try:
+                value = parameter.value  # if parameter is not set yet, this will raise ValueError
+            except ValueError:
+                pass  # parameter is not set yet
```

Comment:
Changed the way this is done. Now it uses `parameter.default` for initialization.